lib/msf/base/sessions/mysql.rb

```diff
@@ -9,6 +9,13 @@ class Msf::Sessions::MySQL < Msf::Sessions::Sql
   def initialize(rstream, opts = {})
     @client = opts.fetch(:client)
     self.console = ::Rex::Post::MySQL::Ui::Console.new(self)
+
+    server_version_vars = @client.query_server_version_vars
```

Comment:
It'd be great to follow a pattern of not having this side effect here if possible 👀

Reply:
The rationale being similar to the last time this approach was used for querying the database name - i.e. instantiating objects shouldn't start make network requests as it can make testing/reusing objects awkward